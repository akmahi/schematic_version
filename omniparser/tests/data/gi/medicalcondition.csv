"ROW1","Domain","Patient First Name","Patient Last Name","Patient Date of Birth","External Patient ID","External Practice ID","Code","Name","Dictionary Name","Start Date","End Date","Resolution Date","Vendor","Source Type"
"1","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","V45.82","Stented coronary artery","ICD9","20150605","","","",""
"2","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","414.01, V45.82","CAD S/P percutaneous coronary angioplasty","ICD9","20160630","","","",""
"3","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","786.2","Cough","ICD9","20221017","","","",""
"4","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","V45.89","S/P cardiac catheterization","ICD9","20200812","","","",""
"5","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","272.4","Hyperlipidemia","ICD9","20150612","","","",""
"6","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","414.00","CAD (coronary artery disease)","ICD9","20160630","","","",""
"7","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","786.50","Chest pain","ICD9","20180710","","","",""
"8","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","786.09","Dyspnea on exertion","ICD9","20221017","","","",""
"9","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","V45.89","S/P left heart catheterization by percutaneous approach","ICD9","20200812","","","",""
"10","Medical Conditions","LLOYD","AARON","19721219","2761135","Epic","401.9","Essential hypertension","ICD9","20150507","","","",""
