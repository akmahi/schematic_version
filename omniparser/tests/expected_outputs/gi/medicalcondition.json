[
	{
		"medical_condition": {
			"code": "V45.82",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "Stented coronary artery",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "1",
			"start_date": "20150605"
		}
	},
	{
		"medical_condition": {
			"code": "414.01, V45.82",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "CAD S/P percutaneous coronary angioplasty",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "2",
			"start_date": "20160630"
		}
	},
	{
		"medical_condition": {
			"code": "786.2",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "Cough",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "3",
			"start_date": "20221017"
		}
	},
	{
		"medical_condition": {
			"code": "V45.89",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "S/P cardiac catheterization",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "4",
			"start_date": "20200812"
		}
	},
	{
		"medical_condition": {
			"code": "272.4",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "Hyperlipidemia",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "5",
			"start_date": "20150612"
		}
	},
	{
		"medical_condition": {
			"code": "414.00",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "CAD (coronary artery disease)",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "6",
			"start_date": "20160630"
		}
	},
	{
		"medical_condition": {
			"code": "786.50",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "Chest pain",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "7",
			"start_date": "20180710"
		}
	},
	{
		"medical_condition": {
			"code": "786.09",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "Dyspnea on exertion",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "8",
			"start_date": "20221017"
		}
	},
	{
		"medical_condition": {
			"code": "V45.89",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "S/P left heart catheterization by percutaneous approach",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "9",
			"start_date": "20200812"
		}
	},
	{
		"medical_condition": {
			"code": "401.9",
			"dictionary": "ICD9",
			"domain": "Medical Conditions",
			"external": {
				"external_id": "Epic"
			},
			"name": "Essential hypertension",
			"patient": {
				"date_of_birth": "19721219",
				"external_id": "2761135",
				"first_name": "LLOYD",
				"last_name": "AARON"
			},
			"row": "10",
			"start_date": "20150507"
		}
	}
]
